webapp/src/views/nominate.vue

```diff
@@ -171,12 +168,8 @@ export default {
         }
 
         await this.nominate(data);
-        
-        if (this.message.type === 'error') {
-          this.validationMessages = [this.message.message, this.message.message, this.message.message];
-          this.validationResults = [false, false, false];
-        }
-        else {
+
+        if (this.message.type !== 'error') {
           this.validationResults = [null, null];
           this.$bvModal.show('nominate-success');
         }
```

Comment:
When I tested adding a nominee, the success dialog did not appear, even though the request was successful.